src/main/java/com/kdt_y_be_toy_project2/domain/trip/dto/CreateTripRequest.java

```diff
@@ -0,0 +1,27 @@
+package com.kdt_y_be_toy_project2.domain.trip.dto;
+
+import com.kdt_y_be_toy_project2.domain.trip.domain.TripType;
+import jakarta.validation.constraints.NotBlank;
+import jakarta.validation.constraints.NotNull;
+import lombok.Builder;
+import org.springframework.format.annotation.DateTimeFormat;
+
+import java.time.LocalDateTime;
+
+@Builder
+public record CreateTripRequest(
+        @NotBlank
```

Comment:
현재는 개발단계이긴 한데 추가적인 메시지를 보내주는 것도 좋아보이네요!
```suggestion
        @NotBlank(message = "여행명은 필수입니다.")
```